definitions/consented/json/CaseEvent.json

```diff
@@ -675,5 +675,89 @@
     "PostConditionState": "*",
     "SecurityClassification": "Public",
     "ShowEventNotes": "Y"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "FinancialRemedyMVP2",
+    "ID": "FR_awaitingInfo",
+    "Name": "Awaiting Information",
+    "Description": "Awaiting Information",
+    "DisplayOrder": 38,
+    "PreConditionState(s)": "applicationSubmitted;applicationIssued;responseReceived;orderMade;consentOrderApproved",
+    "PostConditionState": "awaitingInfo",
+    "SecurityClassification": "Public",
+    "ShowEventNotes": "Y"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "FinancialRemedyMVP2",
+    "ID": "FR_caseSubmissionFromAwaitingInfo",
+    "Name": "Case Submission",
+    "Description": "Case Submission",
+    "DisplayOrder": 39,
+    "PreConditionState(s)": "awaitingInfo",
+    "PostConditionState": "applicationSubmitted",
+    "SecurityClassification": "Public",
+    "ShowEventNotes": "Y"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "FinancialRemedyMVP2",
+    "ID": "FR_issueApplicationFromAwaitingInfo",
+    "Name": "Issue Application",
+    "Description": "Issue Application",
+    "DisplayOrder": 40,
+    "PreConditionState(s)": "awaitingInfo",
+    "PostConditionState": "applicationIssued",
+    "SecurityClassification": "Public",
+    "ShowEventNotes": "Y"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "FinancialRemedyMVP2",
+    "ID": "FR_responseReceivedFromAwaitingInfo",
```

Comment:
This one, the following event could be re-used: FR_respondToOrder just adding 'awaitingInfo' in the preconditionState

Reply:
Same as above.